src/Workspaces/Core/Portable/Rename/RenameUtilities.cs

```diff
@@ -122,6 +122,15 @@ private static bool ShouldRenameOnlyAffectDeclaringProject(ISymbol symbol)
             var symbolInfo = semanticModel.GetSymbolInfo(token, cancellationToken);
             if (symbolInfo.Symbol != null)
             {
+                // see https://github.com/dotnet/roslyn/issues/10898
+                // we are disabling rename for tuple fields for now
+                // 1) compiler does not return correct location information in these symbols
+                // 2) renaming tuple fields seems a complex enough thing to require some design
+                if (symbolInfo.Symbol.ContainingType?.IsTupleType == true)
+                {
+                    return TokenRenameInfo.NoSymbolsTokenInfo;
```

Comment:
I believe it would be better to put this check in `AbstractEditorInlineRenameService.GetRenameInfo`, since the check there allows you to return a string so you can put a message of "We don't support renaming tuples yet". Otherwise this will make it look like it's a bug that you're going to get feedback on.